spec/price_list_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'price_list'
+
+describe PriceList do
+
+  describe '#display_price_list' do
+    it 'displays the correct list of prices' do
+      expect(subject.display_price_list).to eq PriceList::PRICE_LIST
+    end
+  end
+
+  describe '#select_item' do
+    it 'allows a user to select a certain amount of an item' do
+      expect(subject.list_selection).to include { subject.select_item("Chicken Korma", 3) }
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)